### PR TITLE
doc: added iceberg compaction behavior doc

### PR DESCRIPTION
**Pull request by alokhp**

### CODE REVIEW

File: docs/tutorials/compaction-cow-mor.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+age INT,
+zipcode STRING,
+timestamp TIMESTAMP
+)
+TBLPROPERTIES (
+'write.update.mode' = 'merge-on-read',
+'write.merge.mode' = 'merge-on-read',
+'write.delete.mode' = 'merge-on-read'
+);
+```
```
Comment (garayevs) [inline docs/tutorials/compaction-cow-mor.md:52]: I think this should be formatted - (`Reformat Code` in Intellij).

File: docs/tutorials/compaction-cow-mor.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+## Setup
+### Create a catalog
+
+<Img src="/img/tutorial/compaction-cow-mor/create-catalog.png" alt="Create catalog"/>
+
+:::info To ensure the newly created catalog is detected, you may need to restart the lakehouse if it was already running.
+:::
+
+### Create database and tables
+```
```
Comment (garayevs) [inline docs/tutorials/compaction-cow-mor.md:25]: Plz try to add `sql showLineNumbers` after ```

File: docs/tutorials/compaction-cow-mor.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+### Datafiles
+#### Copy On Write (COW)
+There are currently four files remaining at the file system level. This situation arises from the existence of a snapshot of the update operation, which includes three of these files. The fourth file is the actual compacted file referenced by the latest snapshot. The older files will be eliminated during subsequent compaction runs as we now have newer snapshots available. This process will be illustrated in the following step.
+
+<Img src="/img/tutorial/compaction-cow-mor/compaction-run1-cow-fs.png" alt="compaction-run1-cow-fs"/>
+
+#### Merge On Read (MOR)
+In a manner akin to COW, we maintain a single file that directs to the most recent snapshot; however, the majority of the files are linked to the earliest snapshot generated by the update command.
+
+<Img src="/img/tutorial/compaction-cow-mor/compaction-run1-mor-fs.png" alt="compaction-run1-mor-fs"/>
```
Comment (garayevs) [inline docs/tutorials/compaction-cow-mor.md:159]: It should be possible to make picture dynamic - if background is light mode, then use picture for that, if background is dark mode then use picture for that.

I think @vugardadalov has done something like that for SSO pages - https://iomete.com/resources/user-guide/single-sign-on/okta

File: docs/tutorials/compaction-cow-mor.md (hunk 1 above)
Comment (alokhp) [reply docs/tutorials/compaction-cow-mor.md:53]: Done

File: docs/tutorials/compaction-cow-mor.md (hunk 2 above)
Comment (alokhp) [reply docs/tutorials/compaction-cow-mor.md]: Done

File: docs/tutorials/compaction-cow-mor.md (hunk 3 above)
Comment (alokhp) [reply docs/tutorials/compaction-cow-mor.md:159]: Will check this

**Review by nurlanmamedov — APPROVED** [review]

**Pull request merged**